SR2022/2021-12-10-january-virtual-tech-day.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2022 teams
+subject: January Virtual tech day
+---
+
+Hello,
+
+We will be hosting a [virtual tech day on the 22nd January][event-link] in our [Discord](https://studentrobotics.org/docs/team_admin/discord) server.
```

Comment:
Probably best to mention early-on that we're doing it differently to the last tech day